uni-bern-institut-fur-musikwissenschaft-note.csl

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" names-delimiter=" / " demote-non-dropping-particle="never" default-locale="de">
```

Comment:
while legal as per specs, I'd strongly recommend removing any styling elements on `names`, `bibliograph` or `citation` in `style` -- it's a mess to troubleshoot.

Reply:
Done